src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let discount = 0;
+  const PRICE_PER_DAY = 40;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  if (days >= SHORT_TERM) {
+    discount = SHORT_TERM_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
+    discount = LONG_TERM_DISCOUNT;
+  }
+
+  return PRICE_PER_DAY * days - discount;
```

Comment:
Go from a bigger value to the lowest one. And you can use `early` return
```suggestion
  if (days >= LONG_TERM) {
    return defaultPrice - LONG_TERM_DISCOUNT;
  }

  if (days >= SHORT_TERM) {
    return defaultPrice - SHORT_TERM_DISCOUNT;
  }

  return defaultPrice;
```

Reply:
Not fixed